src/example-types/metricGroups/pivot.js

```diff
@@ -109,22 +109,21 @@ let buildQuery = async (node, schema, getStats) => {
     let sortField = getSortField(sort)
 
     return _.reduce(
-      async (children, group) => {
-        // Calculating subtotal metrics at each group level if not flattening or drilling down
-        // Support for per group stats could also be added here - merge on another stats agg blob to children based on group.stats/statsField or group.values
-        if (!node.flatten && !node.drilldown)
-          children = _.merge(await children, statsAggs)
+      async (children, row) => {
+        // Calculating subtotal metrics at each row level if not drilling down
+        // Support for per row stats could also be added here - merge on another stats agg blob to children based on row.stats/statsField or row.values
+        if (!node.drilldown) children = _.merge(await children, statsAggs)
         // At each level, add a filters bucket agg and nested metric to enable sorting
         // For example, to sort by Sum of Price for 2022, add a filters agg for 2022 and neseted metric for sum of price so we can target it
         // As far as we're aware, there's no way to sort by the nth bucket - but we can simulate that by using filters to create a discrete agg for that bucket
         if (!_.isEmpty(sort)) {
           children = _.merge(sortAgg, await children)
-          // Set `sort` on the group, deferring to each grouping type to handle it
+          // Set `sort` on the row, deferring to each rowing type to handle it
           // The API of `{sort: {field, direction}}` is respected by fieldValues and can be added to others
-          group.sort = { field: sortField, direction: sort.direction }
+          row.sort = { field: sortField, direction: sort.direction }
         }
-        let build = lookupTypeProp(_.identity, 'buildGroupQuery', group.type)
-        return build(group, await children, groupingType, schema, getStats)
+        let build = lookupTypeProp(_.identity, 'buildGroupQuery', row.type)
+        return build(row, await children, groupingType, schema, getStats)
```

Comment:
I'm pretty sure this is another case that should be groups - `buildGroupQuery` here works on both columns and rows

Reply:
Fixed